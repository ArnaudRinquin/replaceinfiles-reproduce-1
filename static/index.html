<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <div>hello world</div>
    hello
    world
  </body>
</html>
