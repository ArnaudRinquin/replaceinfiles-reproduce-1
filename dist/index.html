<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <link href="/main.cdfdc1.css" rel="stylesheet" type='text/css'>
  </head>
  <body>
    <p>1 + 1 is {{1 + 1}}</p>
    <hello-world name="Earth"></hello-world>
    <script src="/bundle.78d67b.js"></script>
  </body>
</html>
