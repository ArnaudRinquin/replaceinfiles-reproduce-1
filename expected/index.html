<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <div>goodbye earth</div>
    goodbye
    earth
  </body>
</html>
